src/main/java/frc/robot/subsystems/HangingSystem.java

```diff
@@ -0,0 +1,66 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.Servo;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+import javax.swing.*;
+
+public class HangingSystem extends SubsystemBase {
+    private final Servo motorR;
+    private final Servo motorL;
+
+
+    public HangingSystem(){
+        motorR =  new Servo(RobotMap.HANGING_MOTOR_RIGHT_ID);
+        motorL =  new Servo(RobotMap.HANGING_MOTOR_LEFT_ID);
+    }
+
+    public void openRight(){
+        motorR.setAngle(RobotMap.HANGING_OPEN_ANGLE);
+    }
+
+    public void openLeft(){
+        motorL.setAngle(RobotMap.HANGING_OPEN_ANGLE);
+    }
+
+    public void closeLeft(){
+        motorL.setAngle(RobotMap.HANGING_CLOSE_ANGLE);
+    }
+
+    public void closeRight(){
+        motorR.setAngle(RobotMap.HANGING_CLOSE_ANGLE);
+    }
+
+    public double getLeftAngle(){
+        return motorL.getAngle();
```

Comment:
Both this and `getRightAngle` are useless. `motor.getAngle`/`motor.getPosition` do not query an actual sensor but just return the last value passed to `setAngle`. 

There is no need to do PIDs on them, just run them according to time and we'll see about providing some sensors in the future.